src/jobs/update-server-count-job.ts

```diff
@@ -12,14 +12,17 @@ let BotSites: BotSite[] = require('../../config/bot-sites.json');
 let Config = require('../../config/config.json');
 let Logs = require('../../lang/logs.json');
 
-export class UpdateServerCountJob implements Job {
+export class UpdateServerCountJob extends Job {
     public name = 'Update Server Count';
     public schedule: string = Config.jobs.updateServerCount.schedule;
     public log: boolean = Config.jobs.updateServerCount.log;
+    public runOnce: boolean = Config.jobs.updateServerCount.runOnce;
+    public initialDelaySeconds: number = Config.jobs.updateServerCount.initialDelaySeconds;
```

Comment:
These aren't needed now.

Reply:
They aren't needed, but seeing as we only have one job I added them to both the job file and the config, to show new users they are able/how to use them